strings/steve.json

```
@@ -15,6 +19,11 @@
         "We should store bank records in a webscale datastore, like /dev/null.",
         "You ever wonder if /dev/null supports sharding?",
         "Do you know who ate all the donuts?",
-        "What if we use a language that was written on a napkin and created over 1 weekend for all of our servers?"
+        "What if we use a language that was written on a napkin and created over 1 weekend for all of our servers?",
+        "What if we took a locker, some random trash, and made an exosuit out of it? Wouldn't that be like, super cool and stuff?",
```

Oops, forgot to put an and here before some random trash
```suggestion
        "What if we took a locker, and some random trash, and made an exosuit out of it? Wouldn't that be like, super cool and stuff?",
```

---

I'm not sure about the comma placements here

maybe it fits, though? idk